Quadriga/src/main/java/edu/asu/spring/quadriga/web/publicwebsite/WebsiteAboutEditController.java

```diff
@@ -0,0 +1,75 @@
+package edu.asu.spring.quadriga.web.publicwebsite;
+
+import java.security.Principal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import edu.asu.spring.quadriga.aspects.annotations.AccessPolicies;
+import edu.asu.spring.quadriga.aspects.annotations.CheckedElementType;
+import edu.asu.spring.quadriga.aspects.annotations.ElementAccessPolicy;
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.publicwebsite.IAboutTextManager;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+import edu.asu.spring.quadriga.web.login.RoleNames;
+
+/**
+ * This controller manages public website's about page of a project. Information
+ * of title and description is editable.
+ *
+ * @author Rajat Aggarwal
+ *
+ */
+
+@Controller
+public class WebsiteAboutEditController {
+
+	@Autowired
+	private IRetrieveProjectManager projectManager;
+
+	@Autowired
+	private IAboutTextManager aboutTextManager;
+
+	@AccessPolicies({ @ElementAccessPolicy(type = CheckedElementType.PROJECT, paramIndex = 0, userRole = {
+			RoleNames.ROLE_PROJ_COLLABORATOR_EDITOR }) })
```

Comment:
Project owners and project admins should be able to change the about text. Right now you let editors do it.